views/game.hbs

```diff
@@ -37,7 +56,7 @@
         {{else}}
         <div class="cell" data-color="{{this.value}}">
         </div>
-        {{/areEqual}}
+        {{/cellCanBePlayed}}
```

Comment:
the template is becomeing hard to read. Time to split it?

Reply:
indeed it would be a great time to do it... we'll see if we got time later on 🙂